src/functions/instance/getTableData.js

```diff
@@ -2,6 +2,20 @@ import describeTable from '../api/instance/describeTable';
 import searchByValue from '../api/instance/searchByValue';
 import searchByConditions from '../api/instance/searchByConditions';
 
+const getAttributesFromTableData = (tableData, existingAttributes) => {
+  const existing = new Set(existingAttributes);
+  const extra = new Set();
```

Comment:
This looks good, and basically is what we want, but I would actually like to be a more adaptive with these columns. Let's make `extra` a Map of the property count, (property name is key, count of records with that property). And then let's sort by count (most first) and only add enough columns to have 8 columns:
`return Array.from(extra).sort(([,a], [,b]) => b - a).map(([key]) => key).slice(0, 8 - existingAttributes.length);`
And drop out of this function at the beginning if there are already eight columns ` if (existingAttributes.length >= 8) return;`